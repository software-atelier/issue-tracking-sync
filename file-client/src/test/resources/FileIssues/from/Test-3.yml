---
key: "Test-3"
summary: "doNotSynchronize field is not changed"
status: "changed"
description: |
  This issue has an field, what not should be synchronized.
  After synchronization all fields except that are synchronized.
doNotSynchronize: "NEW Value to be ignored!"
lastUpdated: 2021-08-27T14:38
attributes: { }
comments: [ ]
attachments: [ ]
