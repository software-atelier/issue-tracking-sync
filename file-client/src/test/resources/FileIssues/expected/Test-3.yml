---
key: "Test-3"
summary: "doNotSynchronize field is not changed"
status: "changed"
description: |
  This issue has an field, what not should be synchronized.
  After synchronization all fields except that are synchronized.
doNotSynchronize: "OLD Value"
lastUpdated: 2021-08-27T14:38
attributes: { }
comments: [ ]
attachments: [ ]
